### PR TITLE
Fix Issues #36, #37 and more

### PR DESCRIPTION
**Pull request by aravinat21**
Fix https://github.com/ThanatosShinji/onnx-tool/issues/36 and https://github.com/ThanatosShinji/onnx-tool/issues/37 and more

### CODE REVIEW

File: onnx_tool/node.py [hunk 1]
```diff
@@ -1626,15 +1626,20 @@ def value_infer(self, intensors: []):
 
     def profile(self, intensors: [], outtensors: []):
         macs = 0
+        print('in', intensors)
+        print('out', outtensors)
```
Comment (ThanatosShinji) [inline onnx_tool/node.py:1630]: remove these debug codes

File: onnx_tool/node.py [hunk 2]
```diff
@@ -1626,15 +1626,20 @@ def value_infer(self, intensors: []):
...
         if len(outtensors) == 1:
             if len(intensors) == 3 or len(intensors) == 2:
                 kernel_shape = _get_shape(intensors[1])
                 outvol = volume(_get_shape(outtensors[0]))
+                print('kernel_shape', kernel_shape)
+                print('outvol', outvol)
```
Comment (ThanatosShinji) [inline onnx_tool/node.py:1636]: remove these print

**Review by ThanatosShinji — CHANGES_REQUESTED** [review]
I think your pr is not ready for review. A lot of debug print here. Let me know when you are ready. Thanks for your contribution!

**Comment by aravinat21** [comment]
done plz check

**Review by ThanatosShinji — APPROVED** [review]

**Pull request merged**